platform-specific/dtls_prng_posix.c

```diff
@@ -37,7 +37,14 @@ int
 dtls_prng(unsigned char *buf, size_t len) {
 #ifdef HAVE_GETRANDOM
   return getrandom(buf, len, 0);
-#else /* !HAVE_GETRANDOM */
+#elif defined(HAVE_RANDOM)
+  size_t klen = len;
+  while (len--)
+    *buf++ = random() & 0xFF;
+  return klen;
```

Comment:
random() gives 31 bits of randomness.  To make this more efficient, I would suggest
````
+  size_t klen = len;
+  u_int16_t r_value = (uint16_t)random();
+  
+  while (len > sizeof(r_value)) {
+    memcpy(dst, &r_value, sizeof(r_value));
+    len -= sizeof(r_value);
+    dst += sizeof(r_value);
+    r_value = (uint16_t)random();
+  }
+  memcpy(dst, &r_value, len);
+  return klen;
````
Otherwise this PR looks good to me.

Reply:
According [19.8.1 ISO C Random Number Functions](https://www.gnu.org/software/libc/manual/html_node/ISO-Random.html), the value range is`0-RAND_MAX` with `RAND_MAX` from 2147483647 to 32767. So, I'm not sure, if that pays off.